onboarder/installation/index.rst

```diff
@@ -89,15 +47,13 @@ Import the extension configuration in the ``app/config/config.yml`` file (after
 Make the credential files accessible to Akeneo PIM software
 -----------------------------------------------------------
 
-In the parameters package the Akeneo team put 3 credential files:
+In the parameters package the Akeneo team put 1 credential file:
```

Comment:
"In the parameters package the Akeneo team put a serviceAccount.json credential file used...